### DISCUSSION TITLE
Activating Discussions

### DISCUSSION THREAD

**Discussion by igordejanovic** [textX/textX#325] [Announcements]
We're activating Discussion as a venue for all questions and discussions about textX and as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

In the following period we'll work on moving all content from Issues that belongs here. Issues will become a place for... well, issues :smiley:

**Comment by Santhoshstark06** [comment]
Hey !I'm Santhosh and let's colabrate and start a community,what do you say @igordejanovic

**Reply by igordejanovic** [reply]
Hi @Santhoshstark06. This is a community for users and contributors of textX project. If you are interested in textX you are welcome. Please check out our [contributing guide](https://github.com/textX/textX/blob/master/CONTRIBUTING.md) to see how you can help.
